### ISSUE TITLE
Use predetermined points in irreducible Brillouin zone interpolation

### ISSUE THREAD

**Issue by g5t** [pace-neutrons/Pace-Project-Plan#83]
Detailed [elsewhere](https://github.com/g5t/brille/issues/14), there are cases where to-be-interpolated data can not be easily calculated at points specified by brille. In order to support such cases, create a new interpolation object that triangulates provided points where the data has been calculated instead of requesting calculations at a list of new points.

**Comment by stfc-lam** [comment]
On hold

**Comment by cmarooney-stfc** [comment]
@g5t HI Greg, could you add a few words to this explaining why it is on hold, please. Thanks Chris

**Comment by g5t** [comment]
The link I provided above to more details broke when I moved the `brille` repository. The correct location where I have detailed current hold-ups [is in the new repository location](https://github.com/brille/brille/issues/14).
As a summary, there are a number of complications which make this a difficult problem to solve generally -- plus the specific motivation for the feature was addressed by other (non-`brille`) developments. 

It is probably safe to remove this from the PACE Project Board and I can leave it as a possible future enhancement within `brille`.
